process_manifest/iiifCanvas.py

```diff
@@ -8,9 +8,8 @@ def __init__(self, manifest, item_data):
         iiifItem.__init__(self, item_data.get('id'), 'Canvas')
         self.manifest = manifest
         self.label = self._lang_wrapper(item_data.get('label', item_data.get('title')))
-        self.height = item_data.get('width', 2000)
-        self.width = item_data.get('height', 2000)
         self.item_data = item_data
+        self._set_width_and_height()
```

Comment:
I'd just go with this in init:
```suggestion
        self.height = item_data.get('height', manifest.config['canvas-default-height'])
        self.width = item_data.get('width', manifest.config['canvas-default-width'])
```

but if you're going to go with what you've got you need to change line 67 to be self.width not self.height

Reply:
this does not work because item_data returns empty string because it has the keys of width and height so it returns empty string and that needs to be tested again.

Reply:
I also changed it to be less duplicative looking